pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_vaccination_data.py

```diff
@@ -293,7 +292,7 @@ def test_sanitizing_based_on_regions(self):
             returndata[returndata['Age_RKI'] == '11+'].reset_index(drop=True),
             test_4.reset_index(drop=True), check_dtype=False)
 
-    def test_extrapolate_age_groups(self):
+    def test_extrapolate_age_groups_vaccinations(self):
```

Comment:
This could also be a general test or is it specific in vaccination functionality?

Reply:
Since the method is generally very specific to vaccinations, I would leave the test here.